firmware/quintuna/RSM/src/io/io_loadCell.cpp

```diff
@@ -0,0 +1,47 @@
+#include "io_loadCell.h"
+#include "hw_adc.h"
+#include "hw_adcs.h"
+
+// TODO: Find actual max and min values
+#define LOADCELL_MINVOLT (0.0f)
+#define LOADCELL_MAXVOLT (0.0f)
+
+namespace io::loadCell
+{
+
+float voltageToMechancialLoad(float voltage)
+{
+    // TODO: find transfer function (no spec sheet so far just this amazon link:
+    // https://caltsensor.com/product/s-type-load-cells-dyly-103/
+    return 0.0;
+}
+
+bool OCSC(Adc adcPin_toCheck)
```

Comment:
```suggestion
static inline bool OCSC(const Adc adcPin_toCheck)
```

Reply:
I would also move it out of the namespace to make it extra clear that it is an internal